services/oc-guytonmodel/src/opencorservice_demo/input_keymap.json

```diff
@@ -0,0 +1,23 @@
+{
+    "starttime": "starttime",
```

Comment:
actually a side thing, but this information is missing from input.json ? are these files or string? I think strings should be already in input.json?